README.md

```diff
@@ -28,14 +28,12 @@ For more on the background and motivation behind rkt, read the original [launch
 [cni]: https://github.com/appc/cni
 [selinux]: Documentation/selinux.md
 [tpm]: Documentation/devel/tpm.md
-[blog-post]: https://coreos.com/blog/rocket
-[oci]: https://github.com/opencontainers/image-spec
-[oci-rkt]: Documentation/proposals/oci.md
-[docker2aci]: https://github.com/appc/docker2aci
+[oci-image-spec]: https://github.com/opencontainers/image-spec
+[rkt-oci]: https://github.com/coreos/rkt/projects/4
 
 ## Project status
 
-rkt's version 1.0 release marks the command line user interface and on-disk data structures as stable and reliable for external development. Any major changes to those primary areas will be clearly communicated, and a formal deprecation process conducted for any retired features. The (optional) API for pod inspection is not yet completely stabilized, but is quite usable, and an excellent area for testing and participation as it matures.
+The rkt v1.x series provides command line user interface and on-disk data structures stability for external development. Any major changes to those primary areas will be clearly communicated, and a formal deprecation process conducted for any retired features. The (optional) API for pod inspection is not yet completely stabilized, but is quite usable, and an excellent area for testing and participation as it matures.
```

Comment:
Note for the future, we should probably clearly document the stability policy for both of those and the process for changing.

The final bit about the api-service (though it's not said in as many words) should probably be reworded given we're planning to deprecate it.